src/Knp/Menu/Provider/MenuProviderInterface.php

```diff
@@ -9,7 +9,7 @@ interface MenuProviderInterface
     /**
      * Retrieves a menu by its name
      *
-     * @param array $options
+     * @phpstan-param array<string, callable|array{\Closure, string}> $options
```

Comment:
this is wrong. Options are an `array<string, mixed>`. The type you used here was the one I asked you to use for the LazyProvider `$builders`